systemtests/config.py

```diff
@@ -69,14 +70,16 @@ class Configuration:
         proxy_url="https://devnet-gateway.multiversx.com",
         activation_epoch_sirius=629,
         activation_epoch_spica=2327,
-        activation_epoch_relayed_v3=4294967295,
+        activation_epoch_relayed_v3=2991,
```

Comment:
The initial flag (before the last-time fix), but not extremely relevant (only relevant within system tests, for generating sample transactions).